crates/telio-nurse/src/qos.rs

```diff
@@ -203,8 +232,10 @@ impl Analytics {
     ///
     /// A new `Analytics` instance with the given configuration but with no nodes.
     pub fn new(config: QoSConfig, io: Io) -> Self {
+        let (ping_channel_tx, ping_channel_rx) = mpsc::channel(16);
```

Comment:
As this channel won't be used I can reduce the buffer size (or even wrap it in an Option 🤔)

Reply:
This is actually a semaphore count, not the actual buffer size as far as I know

Reply:
wdym by semaphore count? 🤔 

https://docs.rs/tokio/latest/tokio/sync/mpsc/fn.channel.html

Reply:
https://docs.rs/tokio/latest/src/tokio/sync/mpsc/bounded.rs.html#144-156